lib/encoder.py

```diff
@@ -731,10 +729,15 @@ def __init__(self, context):
   def WorkDir(self):
     return self.workdir
 
+  def SearchPath(self):
+    path = [self.workdir]
+    path.extend(['%s/%s' % (this_path, self.context.codec.name)
```

Comment:
Instead of hard-coding /, use os.path.join. In this case, 

path.extend([os.path.join(this_path, self.context.codec.name)]) ...